azure/storagemanager/azure_test.go

```diff
@@ -269,8 +270,8 @@ func storageDistribution(t *testing.T) {
 			response: &cloudops.StorageDistributionResponse{
 				InstanceStorage: []*cloudops.StoragePoolSpec{
 					&cloudops.StoragePoolSpec{
-						DriveCapacityGiB: 111,
-						DriveType:        "Standard_LRS",
+						DriveCapacityGiB: 128,
+						DriveType:        "StandardSSD_LRS",
```

Comment:
Modify the test to pass in a DriveType and expect the same drive type. The same configuration was being handled in TestCase 2